_posts/2025-05-17-screengrab_panel_pointrelease.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Release ScreenGrab 3.0 and lxqt-panel 2.2.1
+slug: screengrab-3-0-panel-2-2-1-release
+date: 2025-05-17 05:18
+promoted: true
+categories: release
+---
+
+The LXQt team announces point releases of the following components:
```

Comment:
I suppose 3.0 will not be a point release.  :]

Reply:
> I suppose 3.0 will not be a point release. :]

Good point :) Yes, here, ScreenGrab is an exception to the rule.

Reply:
My template is like that ;)
Tried other wording now.